### ISSUE TITLE
Join Us page enhancements -

### ISSUE THREAD

**Issue by Aterfax** [SLUGSoc/website-old#51]
User feedback:

> I'm ambivalent about "join us" taking you straight to the uni page without explaining on our own page what joining us entails or why they should do so?

**Comment by boardfish** [comment]
I think the link should definitely stay, but with either more clarity around what it does or an intermediate page saying something along the lines of "we're taking you to the university site so you can pay 3 quid and sign up", obviously more formally. (It's getting late.)

**Comment by Aterfax** [comment]
Yeah I think the feedback was that effectively we should have an intermediate page to convince them what and why they should become one.

**Comment by Aterfax** [comment]
A concise list of benefits of being a member should also be added as an intermediate page before redirecting to the join up SU page.
